### ISSUE TITLE
Service Account naming error in 6.0

### ISSUE THREAD

**Issue by Dejan992** [aquasecurity/aqua-helm#241]
In `aqua-helm/enforcer/templates/serviceaccount.yaml` on line 5 `name: {{ .Release.Name }}-sa` name for the service account is set to be the release name-sa. Where in `aqua-helm/enforcer/templates/enforcer-daemonset.yaml` on line 28 `serviceAccount: {{ .Release.Namespace }}-sa` service account name is set to be namespace-sa. This is causing an error and stopping deployments because release name can be different from namespace. In our case release name is ` enforcer-1617383540` and service account created has name ` enforcer-1617383540-sa`. But enforcer-daemonset.yaml file is looking for `aqua-sa` service account because the namespace we use for this deployment is Aqua.

**Comment by KoppulaRajender** [comment]
@Dejan992, thanks for reporting, this was fixed in the 5.3 version(#196). Now raised fix for 6.0 https://github.com/aquasecurity/aqua-helm/pull/243.

**Comment by rumstead** [comment]
@KoppulaRajender has this change been pushed to the private aqua sec helm repo? I don't see this change in the enforcer chart for 6.2.3 or 6.0.3. 

```
$ helm pull aqua-helm/enforcer --version 6.2.3
$ tar -xvf enforcer-6.2.3.tgz
$ grep serviceAc enforcer-daemonset.yaml                              
      serviceAccount: {{ .Release.Namespace }}-sa

$ helm pull aqua-helm/enforcer --version 6.0.3 && tar -xvf enforcer-6.0.3.tgz && grep 'serviceAccount' 
      serviceAccount: {{ .Release.Namespace }}-sa
```
